### DISCUSSION TITLE
Unknown characters when copying

### DISCUSSION THREAD

**Discussion by locus134** [pawelsalawa/sqlitestudio#3868] [Q&A]
### Details
Copying content will add an invisible unknown character before the first character. When working with other software (such as Excel), it is easy to cause the problem of recognition errors due to unknown characters.

### Operating system
MacOS Catalina 10.15.7

### SQLiteStudio version
Mac 3.2.1

**Comment by locus134** [comment]
When copying to other software, the first character is an invisible unknown character

**Comment by pawelsalawa** [comment]
Are you copying from a data grid, or a code from SQL Editor, or something else?
